packer/install-dependencies.sh

```diff
@@ -18,6 +18,7 @@ installDeps
 if [[ ${FEATURE_FLAGS} == *"docker-engine"* ]]; then
     DOCKER_ENGINE_REPO="https://apt.dockerproject.org/repo"
     installDockerEngine
+    configureDockerEngine
```

Comment:
I believe we also need to call this in customscript for clusters not using VHD (eg. gov cloud)

Reply:
But we already configure docker to use overlay2 in case of CSE scenarios, this change is during VHD builds only

Reply:
where do we do "we already configure docker to use overlay2 in case of CSE scenarios"?

Reply:
https://github.com/Azure/aks-engine/blob/124379b05ed2f2183aeb5eec38b2392d97dc860c/parts/k8s/kubernetesagentcustomdata.yml#L119

Reply:
custom data applies to both VHD and non-VHD scenarios. Is the issue here that we pre-pull images in the VHD script before the storage driver is set to overlay2?

Reply:
Yes thats correct.